cloudsensei/es_operations.py

```diff
@@ -0,0 +1,44 @@
+import os
+from datetime import datetime
+
+from elasticsearch import Elasticsearch
+
+
+class ESOperations:
```

Comment:
You should add cloud-governance package to requirements.txt and import ESOperations